assets/src/settings-page/paired-url-structure.js

```diff
@@ -88,6 +88,173 @@ function getCustomPairedStructureSources( sources ) {
+								)
+							}
+							{
+								entity.title && (
+									': ' + entity.title
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %d is entity ID */
+									sprintf( __( '(ID: %d)', 'amp' ), entity.id )
+								}
+							</small>
+						</li>
+					) )
+				) }
+
+				{ conflicts.terms && (
+					conflicts.terms.map( ( entity ) => (
+						<li key={ `term-${ entity.id }` }>
+							{
+								entity.edit_link ? (
+									<a href={ entity.edit_link } target="_blank" rel="noreferrer">
+										{ entity.label || entity.post_type }
+									</a>
+								) : (
+									entity.label || entity.post_type
+								)
+							}
+							{
+								entity.name && (
+									': ' + entity.name
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %d is entity ID */
+									sprintf( __( '(ID: %d)', 'amp' ), entity.id )
+								}
+							</small>
+						</li>
+					) )
+				) }
+
+				{
+					conflicts.user && (
+						[ conflicts.user ].map( ( entity ) => (
+							<li key={ `user-${ entity.id }` }>
+								{
+									entity.edit_link ? (
+										<a href={ entity.edit_link } target="_blank" rel="noreferrer">
+											{ __( 'User', 'amp' ) }
+										</a>
+									) : (
+										__( 'User', 'amp' )
+									)
+								}
+								{ ': ' + entity.name }
+								{ ' ' }
+								<small>
+									{
+										/* translators: %d is entity ID */
+										sprintf( __( '(ID: %d)', 'amp' ), entity.id )
+									}
+								</small>
+							</li>
+						) )
+					)
+				}
+
+				{
+					conflicts.post_type && (
+						<li key="post_type">
+							{
+								sprintf(
+									/* translators: %s is post type label */
+									__( 'Post type: %s', 'amp' ),
+									conflicts.post_type.label,
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %s is entity name */
+									sprintf( __( '(name: %s)', 'amp' ), conflicts.post_type.name )
+								}
+							</small>
+						</li>
+					)
+				}
+
+				{
+					conflicts.taxonomy && (
+						<li key="taxonomy">
+							{
+								sprintf(
+									/* translators: %s is taxonomy label */
+									__( 'Taxonomy: %s', 'amp' ),
+									conflicts.taxonomy.label,
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %s is entity name */
+									sprintf( __( '(name: %s)', 'amp' ), conflicts.taxonomy.name )
+								}
+							</small>
+						</li>
+					)
+				}
+
+				{ conflicts.rewrite && (
+					<li key="rewrite">
+						{ __( 'Rewrite rules: ', 'amp' ) }
+						{
+							conflicts.rewrite
+								.map( ( entity ) => (
+									<code key={ entity }>
+										{ entity }
+									</code>
+								) )
+								.reduce( ( prev, curr ) => [ prev, ', ', curr ] )
+						}
+					</li>
+				) }
+			</ul>
+		</AMPNotice>
+	);
+}
+SlugConflictsNotice.propTypes = {
+	slug: PropTypes.string.isRequired,
+	conflicts: PropTypes.object.isRequired,
```

Comment:
Could specify the shape of the object here:

```suggestion
	conflicts: PropTypes.shape( {
		post_type: PropTypes.object,
		posts: PropTypes.arrayOf( PropTypes.object ),
		rewrite: PropTypes.arrayOf( PropTypes.string ),
		taxonomy: PropTypes.object,
		terms: PropTypes.arrayOf( PropTypes.object ),
		user: PropTypes.object,
	} ).isRequired,
```